packages/web/src/components/layouts/HomeNew/HomeNew.tsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import Hero from '../../ui/Hero/Hero';
+import Container from 'components/ui/Container/Container';
+import CardList from '../../ui/CardList/CardList';
+import CardGroups from '../../ui/CardGroups/CardGroups';
+import Footer from '../../ui/Footer/Footer';
+import CardGroup from '../../ui/CardGroup/CardGroup';
+
+export type HomeNewProps = {
+  suggestions: string[];
+};
+
+export default function HomeNew({ suggestions }: HomeNewProps) {
+  const popularCards = [
```

Comment:
I would rather define constants inside storybook stories files and pass them as component properties. 

Any component should not contain any hard-coded constants.

Reply:
Seems like this is the place where data fetching is supposed to happen, I'd just leave it as-is and revisit once the PR is past draft stage.